### PR TITLE
Fix merge conflicts on message search

### PR DESCRIPTION
**Pull request by Mstrodl**
Fixes conflicts for message search and adds a small tweak to make jump to message a little easier when it's added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Emacs/Vim turds
+.\#*\#
+\#*\#
+*~
+.tern-port
+
```
Comment (MaxLeiter) [inline .gitignore:6]: These probably belong in your global gitignore

Comment (Mstrodl) [reply .gitignore:6]: Fair point, sorry

**Comment by Mstrodl** [comment]
@MaxLeiter fixed that issue

CC-ing @richrd as well, since it's yours

**Comment by richrd** [comment]
> @MaxLeiter fixed that issue
> 
> CC-ing @richrd as well, since it's yours

Hi! Thanks a lot for participating, and for the rebase effort! I want to go through this in detail before I merge, and the original PR itself also needs some work which needs to be addressed separately. Anyway what specifically are the tweaks you did for "jump to message" to be easier to implement? Also are you on IRC at #thelounge on Freenode or otherwise reachable on IRC? 

Thanks again!

**Comment by Mstrodl** [comment]
@richrd I just added a `focused` property to the URL query parameter. It doesn't jump or anything, but it highlights the message of the specified ID.

**Comment by fnutt** [comment]
@richrd Any progress on the reviewing, please?

**Comment by MaxLeiter** [comment]
Closing in favor of #4197. Thanks for the effort, @Mstrodl!

**Pull request closed without merging**